app/helpers/orchid/application_helper.rb

```diff
@@ -1,5 +1,33 @@
 module Orchid::ApplicationHelper
 
+  def active?(value, comparison=nil)
+    if value.class == Regexp
+      if comparison.nil?
+        # Inform if tried to pass only a Regexp to active?
+        raise "current_page? does not support Regexp comparisons"
+      elsif comparison.class == Symbol
+        value.match(params[comparison]) ? "active" : ""
+      else
+        value.match(comparison) ? "active" : ""
+      end
+    else
+      Array(value).each do |v|
+        begin
+          if comparison.nil?
+            return "active" if current_page? v
+          elsif comparison.class == Symbol
+            return "active" if current_page? comparison => v
```

Comment:
While looking up `current_page?`, I found that the api dock documentation uses ( ).  I mention it only because I'm annoyed by the lack of clarity in rails standards.

I had some trouble mentally figuring out what `current_page? comparison => v` was doing, but functionally it is `current_page?({comparison => v})`, right?  Morning brain...

Reply:
Yeah, I realize APIdock isn't the actual rails documentation, but it tends to come up higher in search results so I click it.  :)